backend/seeders/20220831071727-User.js

```diff
@@ -1,8 +1,10 @@
 "use strict";
 
+const { User } = require("../models");
+
 module.exports = {
   async up(queryInterface, Sequelize) {
-    return queryInterface.bulkInsert("Users", [
+    return await User.bulkCreate([
```

Comment:
- goal is to update if there is already existing else insert if not. I suggested before to use `bulkCreate` instead of `upsert`(update or insert) since sequelize does not have one. 
- But they do have one by using `bulkCreate` with `updateOnDuplicate` clause. (see sample implementation below)
- apply to other seeder
![image](https://user-images.githubusercontent.com/73863685/193507576-7a7ce5d6-e5aa-4b01-90d8-eb88dd32dcf5.png)